region/maridia/outer/Fish Tank.json

```diff
@@ -360,15 +346,10 @@
           "Wave",
           "Spazer",
           "Plasma"
-        ]},
-        {"adjacentRunway": {
-          "fromNode": 1,
-          "usedTiles": 1.5,
-          "inRoomPath": [1, 5],
-          "physics": ["air"]
-        }}
+        ]}
       ],
       "note": [
+        "Requires 4 tiles of run speed (with no open end) to make it past the overhang above the door.",
```

Comment:
The note wasn't updated, here and below.  
I needed 3 tiles no open end.  Any less wasn't enough height or distance.